docs/guide/custom-login-requirements.md

```diff
@@ -34,9 +34,10 @@ While [most password complexity rules are often counterproductive](https://arste
 
 Apart from password complexity, most enhancements to the login form involve at least some new UI code. UI components added this way are integrated into the existing login interface. This involves both backend Node.js development and frontend Vue 2 [component](https://vuejs.org/v2/guide/components.html) development.
 
-Apostrophe allows you to add extra login requirements during two different phases of the login process:
+Apostrophe allows you to add extra login requirements during three different phases of the login process:
 
-* `beforeSubmit`: Requirements in this phase are displayed before the user clicks the Login button. If the requirement has a visible UI, it appears at the bottom of the login form itself. Best used for requirements that have no visible interface but need to monitor user behavior on the form to determine if the user is "real," like [reCAPTCHA v3](https://developers.google.com/recaptcha/docs/v3).
+* `beforeSubmit`: Requirements in this phase are displayed before the user clicks the Login button. If the requirement has a visible UI, it appears at the bottom of the login form itself. Best used for requirements that have no visible interface but need to monitor user behavior on the form to determine if the user is "real," like [hCaptcha](https://docs.hcaptcha.com).
+* `uponSubmit`: Requirements in this phase are displayed after the user clicks the Login button, but before any password verification takes place. Best used for requirements that must re-run each time the user click on the Login button like [reCAPTCHA v3](https://developers.google.com/recaptcha/docs/v3).
```

Comment:
click on -> clicks on

Also include the "best used for requirements that have no visible interface..." text because that's true here too